### DISCUSSION TITLE
Django

### DISCUSSION THREAD

**Discussion by shivam1shroff** [WebMetadataRetrieval/15-Web_Metadata_Retrieval#31] [Ideas]
People interested in learning Django 
Here is the link:- https://www.youtube.com/watch?v=F5mRW0jo-U4
This whole video of 4 hours will help you learn Django completely

**Comment by devsoftzz** [comment]
Nice Resource @shivam1shroff , He has really good content for Django on his youtube channel.
Also, Anyone wants to learn about building custom user models along with Login, Signup, Password Change/Reset, Refer https://codingwithmitch.com/courses/building-a-website-django-python/. He has explained it very well.
